Lib/fontTools/otlLib/builder.py

```diff
@@ -2574,7 +2574,8 @@ def add(self, glyphs):
             return
         self.classes_.add(glyphs)
         for glyph in glyphs:
-            assert glyph not in self.glyphs_
+            if glyph in self.glyphs_:
+                raise TypeError(f"Glyph {glyph} already present in class.")
```

Comment:
I don't think TypeError is the most appropriate, maybe ValueError? Or a fontTools-specific error class? @madig what do you think?